Xplat/src/main/java/vazkii/botania/common/helper/ForcePusher.java

```diff
@@ -0,0 +1,9 @@
+package vazkii.botania.common.helper;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.world.level.Level;
+
+public interface ForcePusher {
```

Comment:
Usually, even if we have an existing Mixin, we make a separate Accessor mixin to access internal methods. Can you just make this an Accessor instead of a custom interface that's mixed in?